### PR TITLE
Update navbar to match style documents, closes #552

### LINKED ISSUE #552: new navbar

### PR DESCRIPTION
**Pull request by hackmastera**

### CODE REVIEW

File: config/locales/blacklight.en.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 en:
   blacklight:
-    application_name: 'CHF Hydra'
+    application_name: 'Digital Collections'
```
Comment (jrochkind) [inline config/locales/blacklight.en.yml]: Do we maybe want this to say "CHF Digital Collections"? It shows up in places like email subjects and stuff, as well as HTML `<titles>` (and from that: bookmarks, facebook shares, etc). I suppose one could even argue for "Chemical Heritage Foundation Digital Collections", I dunno.

File: config/locales/sufia.en.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ en:
   curation_concerns:
     institution_name:   "CHF"
   sufia:
-    product_name:       "CHF Digital Collections (Beta)"
+    product_name:       "Digital Collections"
```
Comment (jrochkind) [inline config/locales/sufia.en.yml]: Same, if above in i18n is changed, this should probably match, I don't really know where one is used vs the other, matching seems a good bet.

Comment (hackmastera) [inline config/locales/sufia.en.yml]: Looks like there's a helper that overrides application_name with the value of product_name.

**Comment by hackmastera** [comment]
updated translations and logo partial; please re-review

**Comment by jrochkind** [comment]
looks good to me, not sure about travis failure?

**Comment by hackmastera** [comment]
oops, typo. thanks travis. will fix.

**Pull request merged**